.github/workflows/python-ci.yml

```diff
@@ -16,12 +16,17 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install system dependencies
       env:
         PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
```

Comment:
I believe the index url should be in the env for the pip installation section rather than here.